### PR TITLE
Show list of generations: dashboard

### LINKED ISSUE #464: Show list of generations: dashboard
Child of #392

### PR DESCRIPTION
**Pull request by artem-rozhkov-dev-optima**
close #464

### CODE REVIEW

File: src/_designs/screens/dashboard/index.vue [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
     </PageHeader>
-    <Generations>
-      <RouterLink to="generations/generation">
-        <Generation :data="data" />
-      </RouterLink>
-      <Generation :data="data" />
-      <Generation :data="data" />
-      <Generation :data="data" />
+    <Generations v-if="generations">
+      <!--<RouterLink to="generations/generation">-->
```
Comment (ror-shubham) [inline src/_designs/screens/dashboard/index.vue]: Please remove the commented out code

File: src/_designs/components/generation/index.vue [hunk 2]
```diff
@@ -43,7 +40,7 @@
           title="Total"
         >
           <FormatAeUnit
-            :value="data.key_block.total"
+            :value="data.keyBlock.target"
```
Comment (ror-shubham) [inline src/_designs/components/generation/index.vue]: The data (total, reward) is not available as of now. We should just not show the wrong data in its place. We can add the fields when they become available.
For now, I'd recommend removing 'total', 'rewards' fields altogether.

File: src/_designs/components/generation/index.vue [hunk 3]
```diff
@@ -8,11 +8,11 @@
...
           />
         </div>
-        <BlockHeight :value="data.key_block.height" />
+        <BlockHeight :value="data.keyBlock.height" />
       </div>
       <div class="container-first-inner">
         <Account
-          :value="data.key_block.beneficiary"
+          :value="data.keyBlock.beneficiary"
           title="Sender"
```
Comment (ror-shubham) [inline src/_designs/components/generation/index.vue]: The title will be `Beneficiary`, not `Sender`

File: src/_designs/screens/dashboard/index.vue (hunk 1 above)
Comment (artem-rozhkov-dev-optima) [reply src/_designs/screens/dashboard/index.vue]: updated

File: src/_designs/components/generation/index.vue (hunk 2 above)
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/generation/index.vue]: updated

File: src/_designs/components/generation/index.vue (hunk 3 above)
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/generation/index.vue]: updated

**Pull request merged**